ros2run/ros2run/command/run.py

```diff
@@ -45,7 +45,7 @@ def main(self, *, parser, args):
         with NodeStrategy(args) as node:
             try:
                 path = get_executable_path(
-                    node=node, package_name=args.package_name,
+                    package_name=args.package_name,
```

Comment:
If you're removing node here, doesn't the `with NodeStrategy(args) as node:` above this become unused?